src/Lykke.Service.PayInternal.Services/CalculationService.cs

```diff
@@ -159,8 +160,11 @@ public class CalculationService : ICalculationService
             PriceCalculationMethod priceValueType,
             IMarkup merchantMarkup)
         {
-            _log.WriteInfoAsync(nameof(CalculationService), nameof(CalculatePrice), new {askPrice, bidPrice}.ToJson(),
-                "Rate calculation").GetAwaiter().GetResult();
+            _log.Info("Rate calculation", new
```

Comment:
Alexey Novicikhin usually suggests to save in context only data wich is used for searching by fields in kibana.